Rakefile

```diff
@@ -1,36 +1,52 @@
 require "logstash/devutils/rake"
 
-ECS_VERSIONS = {
-    v1: 'v1.10.0', # WARNING: v1.11 breaks 6.x (see: https://github.com/elastic/ecs/issues/1649)
+task :'vendor-ecs-schemata' do
+  download_ecs_schemata(:v1, elasticsearch_major: 6, ecs_release_tag: 'v1.10.0') # WARNING: v1.11 breaks 6.x (see: https://github.com/elastic/ecs/issues/1649)
+  download_ecs_schemata(:v1, elasticsearch_major: 7, ecs_release_tag: 'v1.12.1')
+  download_ecs_schemata(:v1, elasticsearch_major: 8, ecs_release_tag: 'v1.12.1', generated_for: 7) { |template| transform_for_es8!(template) }
+
+  # PRERELEASE: 8.0 branch
+  # when pinning to released tag, remove BETA warning.
+  download_ecs_schemata(:v8, elasticsearch_major: 7, ecs_release_tag: '8.0') { |template| transform_for_es7!(template) if template.include?("template") }
+  download_ecs_schemata(:v8, elasticsearch_major: 8, ecs_release_tag: '8.0') { |template| transform_for_es8!(template) unless template.include?("template") }
```

Comment:
The in-block conditionals are meant to make this draft PR resilient to upstream changes in ECS, so that it will work both before and after they migrate ES7-style templates to ES8-style.

Reply:
Upstream ECS has clarified that they will be generating monolith templates only for the _legacy_ template API, even after stack alignment.